src/CheckoutSdk/Sessions/NextAction.cs

```diff
@@ -11,7 +11,7 @@ public enum NextAction
         IssueFingerprint,
 
         [EnumMember(Value = "challenge_cardholder")]
-        ChallengeCardHolder,
+        ChallengeCardholder,
```

Comment:
@armando-ibarra-cko in this case this fix is correct, because the value is challenge_cardholder and not challenge_card_holder